app/models/self_inscription.rb

```diff
@@ -1,113 +0,0 @@
-# frozen_string_literal: true
-
-#  Copyright (c) 2023, Schweizer Alpen-Club. This file is part of
-#  hitobito_sac_cas and licensed under the Affero General Public License version 3
-#  or later. See the COPYING file at the top-level directory or at
-#  https://github.com/hitobito/hitobito_sac_cas
-
-class SelfInscription
-  include ActiveModel::Model
-  include ActiveModel::Attributes
-  include FutureRole::FormHandling
-
-  attr_accessor :person, :group
-  attribute :register_on, :string, default: "now"
-  attribute :register_as, :string, default: "replace"
-
-  delegate :beitragskategorie_label, to: :role
-
-  validates :person, :group, :register_on, :register_as, presence: true
-  validates :register_on, inclusion: {in: :register_on_keys, allow_blank: true}
-  validates :register_as, inclusion: {in: :register_as_keys, allow_blank: true}
-
-  def initialize(person:, group:, **opts)
-    super
-  end
-
-  def group_for_title
-    role.class.name.ends_with?("::Neuanmeldung") ? @group.parent : @group
-  end
-
-  def neuanmeldung?
-    @group.is_a?(Group::SektionsNeuanmeldungenSektion) ||
-      @group.is_a?(Group::SektionsNeuanmeldungenNv)
-  end
```

Comment:
I think this change is related to the `features/signup/sektion_spec.rb` spec failure.